graph-subscriptions-api/datasource/src/models.rs

```diff
@@ -18,12 +18,9 @@ pub struct GatewaySubscriptionQueryResult {
     /// `user` field from ticket payload, 0x-prefixed hex
     #[prost(string, tag = "5")]
     pub ticket_user: ::prost::alloc::string::String,
-    /// `signer` field from ticket payload, 0x-prefixed hex
+    /// the ticket payload, JSON CBOR map data as a string
```

Comment:
pedantic: JSON is a string format, being used instead of CBOR.
```suggestion
    /// the ticket payload, JSON map data
```